modules/migration/migration-service/src/main/java/org/wso2/carbon/is/migration/service/v540/migrator/ClaimDataMigrator.java

```diff
@@ -94,7 +94,14 @@ public void migrate() throws MigrationClientException {
 
             // Migrate other tenants.
             List<Tenant> tenants = Utility.getTenants();
+            List<Integer> inactiveTenants = Utility.getInactiveTenants();
+            boolean ignoreForInactiveTenants = isIgnoreForInactiveTenants();
             for (Tenant tenant : tenants) {
+                int tenantId = tenant.getId();
```

Comment:
Should we have this fix only for ClaimDataMigrator module? Aren't there any other modules needed this fix?

Reply:
Currently other module migration are not depend on the tenants. We can change the migration of other modules as an impovement.